src/blocks/ankitbajpai65/style.css

```diff
@@ -0,0 +1,100 @@
+* {
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+}
+
+#mainDiv {
+    /* border: 2px solid red; */
+    height: 100vh;
+    display: flex;
+    flex-direction: column-reverse;
+}
+
+
+.footer {
+    background-color: black;
+    height: 20rem;
+}
+
+.footerDiv {
+    /* border: 2px solid black; */
+    height: 86%;
+    width: 100%;
+    display: flex;
+}
+
+.footerPart {
+    /* border: 2px solid red; */
+    /* background-color: black; */
+    width: 25%;
+    padding-top: 3rem;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    /* justify-content: center; */
```

Comment:
Here

Reply:
Hi there, I removed the comments/outdated code from CSS file as you mentioned. Please check the same.
Thanks.